### PR TITLE
VeterinaryRegistry

### PR DESCRIPTION
**Pull request by vbre**

### CODE REVIEW

File: VeterinaryValeriya/DomesticAnimal.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        Cold,
+        Cancer,
+        Diarrhea,
+        Fracture
+    }
+
+    class DomesticAnimal
+    {
+        public string name
```
Comment (jane-u) [inline VeterinaryValeriya/DomesticAnimal.cs]: it should be either Name (starting from capital letter) or not a property

File: VeterinaryValeriya/DomesticAnimal.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            get; set;
+        }
+
+        public string disease
```
Comment (jane-u) [inline VeterinaryValeriya/DomesticAnimal.cs]: it should be either Disease (starting from capital letter) or not a property

File: VeterinaryValeriya/ClinicRegistry.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VeterinaryValeriya
+{
+    class ClinicRegistry
+    {
+        private static Dictionary<int, DomesticAnimal> registry;
+
+        private static int countOfAnimals = 0;
```
Comment (olga-jane) [inline VeterinaryValeriya/ClinicRegistry.cs]: is duplicating registry.Count

File: VeterinaryValeriya/ClinicRegistry.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        }
+
+        public Tuple<string, string> this[int elem]
+        {
+            get { return Tuple.Create(registry[elem].name, registry[elem].disease); }
+        }
+
+        static ClinicRegistry()
+        {
+            registry = new Dictionary<int, DomesticAnimal>();
```
Comment (jane-u) [inline VeterinaryValeriya/ClinicRegistry.cs]: can be done on declaration. Usually c'tor is used for more complicated initialization.

File: VeterinaryValeriya/ClinicRegistry.cs (hunk 3 above)
Comment (jane-u) [inline VeterinaryValeriya/ClinicRegistry.cs]: it should be better called something like nextUniqueSequentialNumber because used only for this purpose

**Comment by vbre** [comment]
Fixed and improved

**Pull request merged**